docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+# Validating NeXus files
+
+Note: This is a how-to guide for using different tools to validate NeXus files. If you want to learn more about how validation is done in `pynxtools`, please visit the [explanation page](../learn/nexus-validation.md).
+
+# The goal
+
+Use a tool to validate NeXus files to a given set of NeXus definitions:
+
+  1. [FAIRmat](https://fairmat-nfdi.github.io/nexus_definitions/index.html#)
+
+  2. [NIAC](https://manual.nexusformat.org/)
+
+
+# Validation of a .nxs file
+
+The validity of NeXus files is fundamental to ensure FAIR data. Without specific requirements, it is not possible to understand the data. What type of experiment? What Laser Wavelength? Which voltage? What data is represented at all in the table? What is the unit of the value? Which ISO norm does this refer to? Where was this measured? Which year was this measured?
+
+Therefore you have enter all required fields in the NeXus definition. The requirements are set by the community via workshops or at conferences. You can as well comment the NeXus definitions, to initiate or propose changes/additions. Go to the NeXus definition, and sign-up/log-in and give us some feedback (Red boxes in the image. Expand this panel on the left by clicking on the arrow symbol).  
+![image.png](<./attachments/9d748230d54a8059-image.png>)
+
+Oftentimes, there will be errors in a generated NeXus file (be it by hand or automatically): Typos, missing required concepts, missing attributes, using the incorrect datatype or format (e.g., array instad of list, float instead of integer, etc.). Therefore, a validation is required, to ensure that the data you want to share, is FAIR.
+
+The NeXus file is valid if it complies with the respective NeXus application definition.
+
+This validation is done by software.
+
+
+# Validation software
+
+There are right now three tools, which can be used for validation of NeXus files. All are different and have individual advantages or disadvantages:
+
+1. pynxtools 
+
+2. cnxvalidate
+
+3. punx
+
+Open software is usually shared on Github - There you find usually the most accurate information, as documentation sometimes lags behind. There you see a box with folders and files. Below is the content of the README.md file displayed. This usually shows instructions for installation and handling of the software.
+
+Here are the GitHub links for the thee software packages:
+
+[pynxtools](<https://github.com/FAIRmat-NFDI/pynxtools>)
```

Comment:
I switched the order here to reflect the order above

Reply:
thanks